### DISCUSSION TITLE
Service providers…?

### DISCUSSION THREAD

**Discussion by turoczy** [portland-startups/docs#80] [Ideas]
How do folks feel about incorporating a list of service providers (e.g., legal, accounting, design, development) that are focused on supporting startups in Portland?

**Comment by jeffmartens** [comment]
Not a fan, only because it will become a long list of self promotion and then the value is gone.

**Comment by dbarcos-formos** [comment]
[said without my vendor hat on] We have some good firms/individuals we all have worked with. Each one I know has been fairly active in the community by engaging with founders or sponsoring events. I think that is the best way for them to get recognition not to give them a listing.
